src/Initializer/DynamicRupture.h

```diff
@@ -92,19 +92,19 @@ struct seissol::initializers::DynamicRupture {
   Variable<real[numOfPointsPadded][6]> initialStressInFaultCS;
   Variable<real[numOfPointsPadded][6]> nucleationStressInFaultCS;
   Variable<real[ numOfPointsPadded ]> mu;
-  Variable<real[ numOfPointsPadded ]> slip;
-  Variable<real[ numOfPointsPadded ]> slipStrike; // = Slip1
-  Variable<real[ numOfPointsPadded ]> slipDip;    // = Slip2
+  Variable<real[ numOfPointsPadded ]> slipMagnitude;
+  Variable<real[ numOfPointsPadded ]> slip1; // slip in Y-direction
```

Comment:
Change to:
https://github.com/SeisSol/SeisSol/blob/master/src/Numerical_aux/typesdef.f90#L637

Reply:
Don't you think `accumulatedSlipMagnitude` is better, since it is more verbose?

Reply:
I mean to change the comment 
`// slip in Y-direction `
to a comment inspired by the Fortran line I m pointing to (Y direction is wrong)

Reply:
Ah, I see...